spec/features/admin_blocks_a_redirection_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.feature "Admin blocks and unlinks a redirection" do
+  before do
+    page.driver.browser.basic_authorize(correct_username, correct_password)
+  end
```

Comment:
What do you think about making a small helper method and removing the need for `before` altogether?

```rb
def login
  page.driver.browser.basic_authorize(
    ENV.fetch("ADMIN_USERNAME"),
    ENV.fetch("ADMIN_PASSWORD")
  )
end
```

And then of course in each test:

```rb
scenario "blocks and unlinks a redirection" do
  redirection = create(...)

  login
  visit admin_root_path
```